apps/sms/views/shared/js/activity_handler.js

```diff
@@ -142,7 +144,32 @@ var ActivityHandler = {
       return;
     }
 
-    this.toView({ body: dataToShare });
+    return this.toView({ body: dataToShare });
+  },
+
+  /**
+   * Store a draft for this message, and returns a draftId.
+   *
+   * @param {Object} message to save
+   * @param {(String|Attachment|Array.<Attachment>)} [message.body] The data to
+   * add to the message's body.
+   * @param {String} [message.number] The recipient for this message.
+   * @returns {Promise.<Number>} The new draft id.
+   */
+  _storeDraft(message) {
```

Comment:
__nit:__ I'd say we need a bit more clear name manifesting that this method "creates draft from message/message-like and stores it" :) But overall I like the idea of reusing drafts for such things, and I can't think of any cases when it's bad at the moment, will test on device to be more confident though :)

__note:__ it would be nice to have something shared between this method and the one you use in ConversationView, but I can't think of anything that's much better than this, so I'm ok if we don't spend too much time prettifying things here and improve later.